mygithub/src/components/github/Profile.js

```diff
@@ -1,58 +1,59 @@
-import React,{Component} from 'react';
+import React, { Component } from 'react';
 import RepoList from './RepoList.js';
-class Profile extends Component{
-	
-	render(){
-// Nesting profile component in App component
-		return(
+class Profile extends Component {
+
+	render() {
+		// Nesting profile component in App component
+		return (
```

Comment:
It would be a good idea to add a check at the beginning of the `render()` method to ensure that `this.props.userData` is not empty before attempting to access its properties. This will prevent potential errors if the component is rendered before the user data is fetched.